SI_Toolkit_ASF/config_predictors.yml

```diff
@@ -0,0 +1,40 @@
+predictor_name_main: 'I_love_control'
+
+predictors:
+  ### THESE ARE DEFAULT PREDICTORS! DON'T CHANGE THEM, RATHER ADD YOUR PREDICTOR BELOW
+  neural_default:
+    predictor_type: 'neural'
+    model_name: 'GRU-6IN-32H1-32H2-5OUT-0'
+    path_to_model: './SI_Toolkit_ASF/Experiments/Pretrained-RNN-1/Models/'
+    dt: 0.02
+    update_before_predicting: True
+  GP_default:
+    predictor_type: 'GP'
+    model_name:
+    path_to_model:
+    dt: 0.02
+  ODE_default:
+    predictor_type: 'ODE'
+    model_name:
+    dt: 0.02
+    intermediate_steps: 10
+  ODE_TF_default:
+    predictor_type: 'ODE_TF'
+    model_name:
+    dt: 0.02
+    intermediate_steps: 5
+
+  # ADD YOUR PREDICTORS BELOW
+
+  I_love_control:
+    predictor_type: 'neural'  # Possible options are: 'neural', 'GP', 'ODE, 'ODE_TF'
+    model_name: 'GRU-6IN-32H1-32H2-5OUT-0'
+    path_to_model: './SI_Toolkit_ASF/Experiments/Pretrained-RNN-1/Models/'
+    dt: # Matters only for differential network
+    update_before_predicting: True
+
+  I_love_control_too:
+    predictor_type: 'ODE'
+    model_name:
+    dt: 0.02
+    intermediate_steps: 10
```

Comment:
Not seeing any info on how to use predictor_specification here. In config.yml it's sort of clear

Reply:
Added at the top. Is it what you wanted to see?